### ISSUE TITLE
Stream numbers instead of inputs

### ISSUE THREAD

**Issue by baborok** [0m0D1NGR0NALD/transformer-health#2]
Hello! Do you thinks it is possible to change code to stream random numbers in input field ?

**Comment by 0m0D1NGR0NALD** [comment]
Yes it is possible. All you have to do is web scraping to automate the data
entry.
